gbpservice/nfp/orchestrator/modules/service_orchestrator.py

```diff
@@ -64,12 +67,13 @@ def events_init(controller, config, service_orchestrator):
               'CONSUMER_ADD', 'CONSUMER_REMOVE',
               'APPLY_USER_CONFIG_IN_PROGRESS',
               'UPDATE_USER_CONFIG_PREPARING_TO_START',
+              'FINAL_DEVICE_ACTIVE',
               'UPDATE_USER_CONFIG_IN_PROGRESS',
               'UPDATE_USER_CONFIG_STILL_IN_PROGRESS',
               'DELETE_USER_CONFIG_IN_PROGRESS',
               'CONFIG_APPLIED', 'USER_CONFIG_APPLIED', 'USER_CONFIG_DELETED',
               'USER_CONFIG_DELETE_FAILED', 'USER_CONFIG_UPDATE_FAILED',
-              'USER_CONFIG_FAILED']
+              'USER_CONFIG_FAILED', 'CHECK_USER_CONFIG_COMPLETE']
```

Comment:
Why we need this 2 new events FINAL_DEVICE_ACTIVE, CHECK_USER_CONFIG_COMPLETE,
Is there any issue in existing events